### PR TITLE
React Group Project: Space Travelers Hub

### PR DESCRIPTION
**Pull request by LokurasRlz**
# In this branch, We have worked on the following things: 

- Built a web application for a company that provides commercial and scientific space travel services. ✔️ 
- Setup Linters. ✔️
- Followed Linters. ✔️
- Followed GitFlow. ✔️
- Used Ducks Pattern.
- Used React components.. ✔️
- Used React props. ✔️
- Used React Router. ✔️
- Connected React and Redux. ✔️
- Handled events in a React app. ✔️
-  React state was used. ✔️
- Unit tests with React Testing Library were created. ✔️
-  Styles in a React app were used. ✔️
-  Used React life cycle methods. ✔️
- Applied React best practices and language style guides in code. ✔️
- Store, actions, and reducers were used in React.  ✔️
-  React state was used. ✔️
- Code reviews for team members were performed. ✔️
- Webpage Deployed. ✔️

### CODE REVIEW

**Comment by LokurasRlz** [comment]
We don't know how to stop de bot from Netlify. We tried to use it but It didn't work for us.

File: src/components/Rockets/rockets.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const Rocket = ({ rocket }) => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.Rockets);
+  const {
+    id, rocketName, description, rocketImage, reserved,
+  } = rocket;
+  const handleReserve = (e) => {
+    if (!reserved) {
+      dispatch(reserveRocket(+e.target.id));
+      console.log('reserved...', rockets);
```
Comment (FrederickMih) [inline src/components/Rockets/rockets.js:18]: - [ ] Please do not commit console.log to your repo. It really kills performance and can make confidential data to be visible to anyone using the browser tools to look at your website.

Screenshot from 2022-09-29 00-01-23

It's ugly.

kindly remove all console.log lines in your codebase.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Pull Request has a good title and summary :+1: 
- Correct GitHub workflow has been followed :heavy_check_mark: 
- Rockets and Missions pages have been implemented :+1: 
- No Linter error :heavy_check_mark: 

## Required Changes ♻️

- [ ] Let's try to fix these console errors:

Screenshot from 2022-09-29 00-41-22

- [ ] Let's kindly fix the deployment errors before submitting our project again.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @LokurasRlz @Tadessejemal1       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- No linter errors 
- Nice UI design


Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**